scripts/cibuild

```diff
@@ -70,6 +72,7 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
         echo "Building app, airflow..."
         GIT_COMMIT="${GIT_COMMIT}" docker-compose \
             -f "${DIR}/../docker-compose.yml" \
+            -f "${DIR}/../docker-compose.airflow.yml" \
```

Comment:
This task needs `docker-compose.airflow.yml` to build `nginx` because it adds links to `airflow` services and overrides the `extra_hosts` specified by `docker-compose.yml`